Script is not running, restarting...
Traceback (most recent call last):
  File "/home/user/kva-kva/scripts/python/bot_9.py", line 3, in <module>
    import telebot
ModuleNotFoundError: No module named 'telebot'
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
Script is running
